pages/docs/tutorials/native/using-intellij-idea.md

```diff
@@ -1,50 +1,131 @@
+
+   <div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+
+   ```kotlin
+   fun main() {
+       println("Hello, enter your name:")
+       val name = readLine()
+       name?.replace(" ", "")?.let {
+           println("Your name contains ${it.length} letters")
+       } ?: error("Error while reading input from the terminal: the value can't be null.")
+   }
+   ```
+
+   </div>
+
+3. Save the changes and run the application.
+
+   IntelliJ IDEA opens the **Run** tab and shows the output.
+
+4. Enter your name and enjoy the result:
+
+   ![Application output]({{ url_for('tutorial_img', filename='native/using-intellij-idea/native-new-project-intellij-5.png') }})
+
+
+### Count unique letters in your name
+
+1. Open the file `main.kt` in `src/<your_app_name>Main/kotlin`.
+
+2. Count the distinct letters in your name:
+
+   * Declare the new [extension function](https://kotlinlang.org/docs/reference/extensions.html#extension-functions) `countDistinctCharacters()` for `String`:
+     * Convert the name to lower case using the [`toLowerCase()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/to-lower-case.html) function.
+     * Convert the input string to a collection of characters using the [`toList()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/to-list.html) function.
+     * Select only distinct characters in your name using the [`distinct()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/distinct.html) function.
+     * Count distinct characters using the [`count()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/count.html) function.
+   * Use the `countDistinctCharacters()` function to count unique letters in your name.
+
+   <div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+
+   ```kotlin
+   fun String.countDistinctCharacters() = toLowerCase().toList().distinct().count()
+
+   fun main() {
+       println("Hello, enter your name:")
+       val name = readLine()
+       name?.replace(" ", "")?.let {
+           println("Your name contains ${it.length} letters")
+           println("Your name contains ${it.countDistinctCharacters()} unique letters")
+       } ?: error("Error while reading input from the terminal: the value can't be null.")
+   }
+   ```
+
+   </div>
+
+3. Save the changes and run the application.
+
+   IntelliJ IDEA opens the **Run** tab and shows the output.
+
+4. Enter your name and enjoy the result:
+
+   ![Application output]({{ url_for('tutorial_img', filename='native/using-intellij-idea/native-new-project-intellij-6.png') }})
 
-The sample project can serve as the basis for any new project for Kotlin/Native.
 
 ## What's next?
 
-For further information, check out:
+The sample project can serve as the basis for any new project for Kotlin/Native. For further information, check out:
+
+* [Kotlin/Native Gradle plugin](https://kotlinlang.org/docs/reference/native/gradle_plugin.html)
```

Comment:
This is an old page. There is no Kotlin/Native Gradle plugin since Kotlin 1.3.40. We discussed with Native guys and they should remove this page.

Reply:
Changed the whole section.